servicemodules/discord.py

```diff
@@ -123,7 +123,7 @@ async def on_message(message: Union[discord.message.Message, DiscordMessage]) ->
                     response = module.process_message(message)
                 except Exception as e:
                     why_traceback.append(f"There was a(n) {e} asking the {module} module!")
-                    log.error(self.class_name, error=f"Caught error in {module} module!")
+                    log.error(self.class_name, error=f"Caught error in {module} module: {e}")
```

Comment:
I think it might be better to move this line to `self.utils.log_exception` and also print the reconstructed stack trace from there

Reply:
Sounds good